assets/javascripts/kitten/components/cards/crowdfunding-card/styles.js

```diff
@@ -325,4 +272,135 @@ export const StyledCrowdfundingCard = styled(
       flex-shrink: 0;
     }
   }
+
+  .k-CrowdfundingCard__state {
+    position: absolute;
+    top: 0;
+    right: 0;
+    display: none;
+    background-color: ${COLORS.background1};
+    padding: ${COMPONENT_GUTTER};
+    line-height: 1;
+  }
+
+  .k-CrowdfundingCard__widgetState {
+    display: flex;
+    position: absolute;
+    background: ${COLORS.font1};
+    border-radius: ${pxToRem(25)};
+    margin: ${pxToRem(10)} ${pxToRem(10)} 0 0;
+    right: 0;
+    top: 0;
+    padding: ${pxToRem(10)} ${pxToRem(20)};
+  }
+`
+
+const tabletStyles = css`
+  .k-CrowdfundingCard__image__ownerContainer {
+    display: inline-flex;
+    align-items: center;
+  }
+
+  .k-CrowdfundingCard__title {
+    padding: 0 ${COMPONENT_GUTTER};
+  }
+
+  .k-CrowdfundingCard__title__title {
+    font-size: ${stepToRem(2)};
+  }
+
+  .k-CrowdfundingCard__subtitle__container {
+    display: flex;
+    align-items: center;
+    line-height: 1;
+    padding: 0 ${COMPONENT_GUTTER};
+    margin-top: ${pxToRem(10)};
+  }
+
+  .k-CrowdfundingCard__additionalInfo {
+    padding: 0 ${COMPONENT_GUTTER};
+  }
+
+  .k-CrowdfundingCard__informations {
+    margin-top: ${pxToRem(20)};
+    padding: 0 ${COMPONENT_GUTTER};
+  }
+
+  .k-CrowdfundingCard__informations__infoContainer {
+    margin-right: ${pxToRem(20)};
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  .k-CrowdfundingCard__progressBar {
+    padding: 0 ${COMPONENT_GUTTER};
+    margin-bottom: ${pxToRem(20)};
+  }
+
+  .k-CrowdfundingCard__progressBar__percent {
+    display: inherit;
+  }
+
+  &.k-CrowdfundingCardWidget {
+    .k-CrowdfundingCard__title {
+      padding: 0 ${COMPONENT_GUTTER} 0 0;
+    }
+  }
+
+  &.k-CrowdfundingCard--titlesMinHeight .k-CrowdfundingCard__titleAndDesc {
+    min-height: ${pxToRem(75)};
+  }
+
+  .k-CrowdfundingCard__state {
+    display: inline-flex;
+    align-items: center;
+  }
+`
+
+const desktopStyles = css``
+
+export const StyledCrowdfundingCard = styled(
+  ({ imageContainerRatio, forceVersion, ...props }) => <div {...props} />,
+)`
+  ${({ forceVersion }) => {
+    return (() => {
+      switch (forceVersion) {
+        case 'desktop':
+          return css`
+            ${mobileStyles};
+            ${tabletStyles};
+            ${desktopStyles};
+          `
+        case 'tablet':
+          return css`
+            ${mobileStyles};
+            ${tabletStyles};
+          `
+        case 'mobile':
+          return css`
+            ${mobileStyles};
+          `
+        default:
+          return css`
+            ${mobileStyles};
+
+            @media (min-width: ${pxToRem(ScreenConfig.S.min)}) {
+              ${tabletStyles};
+            }
+
+            @media (min-width: ${pxToRem(ScreenConfig.L.min)}) {
+              ${desktopStyles};
+            }
+          `
+      }
+    })()
+  }}
+
+  .k-CrowdfundingCard__image__imageContainer {
+    padding-top: calc(
```

Comment:
Je propose d'utiliser un commentaire pour pouvoir renseigner dès à présent les cas où on pourra remplacer des usages de props par des custom properties, quand on dépréciera IE11. Une simple recherche globale IE11 permettra de débusquer tous ces cas au lieu d'avoir à relire chaque composant un par un.

(et j'utilise les commentaires `/* comment */` parce qu'ils posent moins de problèmes après minification)

```suggestion
    /* When IE11 depreciates, replace prop with CSS custom property */
    padding-top: calc(
```

Reply:
Ca me va 👍